docs/content/preview/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -332,11 +332,28 @@ EXPLAIN output for querying the table from `eu-west-2`:
 (2 rows)
 ```
 
+## Alter tablespace of tables and indexes
+
+Both tables and indexes can be moved to different tablespaces. The tablespace change will immediately reflect in the config of the table/index, however the tablet move by the load balancer happens in the background. 
+While the load balancer is performing the move it is perfectly safe from a correctness perspective to do reads and writes, however some query optimization that happens based on the data location may be off while data is being moved.
+
+```sql
+ALTER TABLE multi_region_table SET TABLESPACE eu_west_tablespace;
+```
+
+```output
+WARNING:  'tablespace_alteration' is a beta feature!
+LINE 1: ALTER TABLE multi_region_table SET TABLESPACE eu_west_...
+                                       ^
+HINT:  To suppress this warning, set the 'ysql_beta_feature_tablespace_alteration' yb-tserver gflag to true.
+(Set 'ysql_beta_features' yb-tserver gflag to true to suppress the warning for all beta features.)
```

Comment:
This is no longer a beta feature. Here's what the new notice looks like, we can include this here instead:
```
yugabyte=# ALTER TABLE bank_transactions_eu SET TABLESPACE eu_central_1_tablespace;
NOTICE:  Data movement for table bank_transactions_eu is successfully initiated.
DETAIL:  Data movement is a long running asynchronous process and can be monitored by checking the tablet placement in http://<YB-Master-host>:7000/tables
ALTER TABLE
```

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/20562/commits/b6623939e056e04feba0e89ee7f5fa8a2ebd3a09